Dockerfile

```diff
@@ -0,0 +1,24 @@
+# Stage 1: Build Stage
+FROM quay.io/jupyter/minimal-notebook as builder
```

Comment:
will need to use the msdlive base notebook if this is going to run on msdlive

Reply:
changed!